src/java/org/apache/cassandra/service/accord/txn/TxnUpdate.java

```diff
@@ -57,7 +56,7 @@
 import static org.apache.cassandra.utils.ByteBufferUtil.serializedSizeWithVIntLength;
 import static org.apache.cassandra.utils.ByteBufferUtil.writeWithVIntLength;
 
-public class TxnUpdate implements Update
```

Comment:
why?  when we don't have multiple conditions, we can safely just use this class

Reply:
I made this change temporarily to make sure `TxnUpdate` is not used anywhere. Anyway, why would we want to use it? I suppose it could complicate the logic as we would have to deal with single and multiple conditions separately